README.md

```diff
@@ -1,31 +1,151 @@
-Libero Article Store
-====================
+[![Libero][Libero logo]][Libero]  
+  
+Article Store
+=============
 
-[![Build Status](https://github.com/libero/article-store/workflows/CI/badge.svg?branch=master)](https://github.com/libero/article-store/actions?query=branch%3Amaster+workflow%3ACI)
+[![Build status][Build badge]][Build]
+[![Open issues][Open issues badge]][Open issues]
+[![Docker pulls][Docker pulls badge]][Docker image]
+[![License][License badge]][License]
+[![Slack][Slack badge]][Libero Community Slack]
 
-Implementation of a [Libero API](https://libero.pub/api).
+This app provides an HTTP API for creating, maintaining and reading articles.
 
-Requirements
+An article is an [RDF graph], where the root node is a [`http://schema.org/Article`][schema:Article]. It can be
+represented in [JSON-LD] as: 
+
+```json
+{
+  "@context": "http://schema.org/",
+  "@id": "http://example.com/my-article-store/articles/1234567890",
+  "@type": "Article",
+  "name": "My article"
+}
+```
+
+The API uses the [Hydra vocabulary] as its [hypermedia format][HATEOAS], and follows the [Libero API specification].
+
+It's written in [TypeScript], uses the [Koa framework][Koa], and various [RDF/JS libraries][RDF/JS].
+
+Articles are persisted in a [PostgreSQL] database.
+
+<details>
+
+<summary>Further reading</summary>
+
+- [Libero API Specification]
+- [RDF 1.1 Primer]
+- [Hydra Core Vocabulary][Hydra vocabulary]
+- [RDF JavaScript Libraries][RDF/JS]
+  - [Data Model Specification][RDF/JS data model]
+  - [Dataset Specification][RDF/JS dataset]
+
+</details>
+
+Table of contents
+-----------------
+
+1. [Installation](#installation)
+2. [Development](#development)
+3. [Contributing](#contributing)
+4. [Getting help](#getting-help)
+5. [License](#license)
+
+Installation
 ------------
 
-- [Docker](https://www.docker.com/)
-- [GNU Bash](https://www.gnu.org/software/bash/)
-- [GNU Make](https://www.gnu.org/software/make/)
-- [Node.js](https://nodejs.org/) (for development)
+The app is published on Docker Hub as [`liberoadmin/article-store`][Docker image].
 
-Running
--------
+As it is still under heavy development, there are not yet tagged releases. However, an image is available for every
+commit.
+
+### Running an Article Store
+
+1. Start a PostgreSQL container by executing:
+
+   ```shell
+   docker run -d --name article-store-database \
+     -e "POSTGRES_DB=article-store" \
+     -e "POSTGRES_USER=user" \
+     postgres:11.5-alpine
+   ```
+
+2. Run the database creation with an ephemeral Article Store container:
+
+   ```shell
+   docker run --rm \
+     --link article-store-database \
```

Comment:
is this easier than pasting here a Docker Compose configuration that uses published images (rather than starting from source)? Declarative is easier to teardown and to port to other contexts, whereas these containers have to be manually cleaned up.

Reply:
These use published images.

Agree that Compose is easier, happy to _also_ include it. (Feel it's useful to explain what's happening too though. And the database setup complicates it.)

Reply:
https://github.com/libero/article-store/pull/261/commits/b32c172c74dc2f30024e619baeaf4588c62b1876